google/cloud/credentials.h

```diff
@@ -203,6 +203,49 @@ struct UnifiedCredentialsOption {
   using Type = std::shared_ptr<Credentials>;
 };
 
+/**
+ * Configures a custom CA (Certificates Authority) certificates file.
+ *
+ * Most applications should use the system's root certificates and should avoid
+ * setting this option unnecessarily. A common exception to this recommendation
+ * are containerized applications. These often deploy without system's root
+ * certificates and need to explicitly configure a root of trust.
+ *
+ * The value of this option should be the name of a file in [PEM format].
```

Comment:
I realize that `CAINFO` is a term that curl uses, and it may be good to follow suit w/ this name. But "CAINFO" sounds pretty ambiguous to me. I didn't realize this was a file path until I saw it used w/ `ifstream` below (and yes, I must have missed this sentence). Would naming this something like `CARootPemOption` be clearer? Or different and bad? WDYT?

Reply:
You are right, changed the name, PTAL.